### DISCUSSION TITLE
Vote to promote @fg-j to maintainer

### DISCUSSION THREAD

**Discussion by ForestEckhardt** [paketo-buildpacks/feedback#95] [Content Team]
@fg-j  has contributed a number of features and updates to  the `paketo-website` and has established a lot of expertise. She also has spearheaded an initiative to overhaul and improve our doc publishing process with Hugo modules. I'd like to propose that she be promoted to a maintainer position to help us guide this portion of the project forward.

@paketo-buildpacks/content-maintainers  can I get a 👍 / 👎 ?

**Comment by ForestEckhardt** [comment]
@paketo-buildpacks/steering-committee Could you please promote @fg-j to the role of maintainer?

**Comment by ryanmoran** [comment]
Done!
